gulpfile.js

```diff
@@ -150,15 +151,13 @@ gulp.task('webpack', () => {
       test: /\.js$/u,
       enforce: 'pre',
       exclude: /(node_modules|\.spec\.js)/u,
-      use: [
-        {
-          loader: 'webpack-strip-block',
-          options: {
-            start: 'DEVBLOCK:START',
-            end: 'DEVBLOCK:END',
-          },
+      use: [{
```

Comment:
fixed the problems with gulpfile.js while trying to fix hound. I give up. I'm just gonna use eslint-loader and fail on first error since we're moving to react rendering 💩